### ISSUE TITLE
Clarification about Twitter/X Archiving

### ISSUE THREAD

**Issue by newsjunkie247** [internetarchive/wayback-machine-webextension#1054]
Can there be some clarification about what the status is with archiving Twitter/X? When you try to save (on the website) it states "We’re currently facing some limitations when it comes to archiving this site. We apologize for any inconvenience this might cause and appreciate your understanding. Please email us at "info@archive.org" if you would like to discuss this more."  Is this being worked on? Still seems important for the historical record....

**Comment by figolambo** [comment]
I reached out to that email address in April and received this reply: 

"Yes, unfortunately X doesn’t allow us to archive their HTML pages at this time – however, you can often find X posts archived as JSON objects by replacing “[x.com](http://x.com/)” with “[twitter.com](http://twitter.com/)” in your URL query. We have no timeline or information on whether or not this will change in the future.

Sorry we don't have better news for you, but please feel free to reach out if you have any other questions!"

Based on this response, I expect that Twitter/X archiving will remain unavailable for the foreseeable future.

**Comment by moogamouth** [comment]
> that's apocalyptic

Indeed. At least archive.today works, but that's less reputable and possibly not as enduring.

**Comment by zararashigal** [comment]
RESPOOOOOOND, DEEEVS, HELLOOOOOO :3

**Comment by zararashigal** [comment]
ANYONE ALIIIIVEEE *RINGDINGDINGDINGDINGDING* HELLOOOOO :3

**Comment by zararashigal** [comment]
BUMP. BUMP. BUMP. :3

**Comment by deddedbyglamour** [comment]
> RESPOOOOOOND, DEEEVS, HELLOOOOOO :3

girl what are they gonna respond to? elon musk hasn't exactly changed his mind on letting his own words get archived to be brought up when he contradicts himself, there won't be a fix.

**Comment by CookieSnippets** [comment]
> RESPOOOOOOND, DEEEVS, HELLOOOOOO :3
> ANYONE ALIIIIVEEE _RINGDINGDINGDINGDINGDING_ HELLOOOOO :3
> BUMP. BUMP. BUMP. :3

1. stop spamming
2. read the response they gave to @figolambo  in second post of this thread.
3. stop spamming
4. complain to Musk
5. stop spamming
